.github/CODEOWNERS

```diff
@@ -77,6 +77,10 @@ chromium_src/**/extensions/**/*_features.json @brave/sec-team
 chromium_src/tools/json_schema_compiler/ @brave/sec-team
 common/extensions/api/ @brave/sec-team
 
+# chromium_src Typescript and CSS overrides
+chromium_src/**/*.css @brave/chromium-src-web-reviewers
+chromium_src/**/*.ts @brave/chromium-src-web-reviewers
+
 # Lit Mangling
 chromium_src/**/*.lit_mangler.ts @fallaciousreasoning
 tools/chromium_src/lit_mangler/ @fallaciousreasoning
```

Comment:
might as well change these to use the new group too (and the first one can be removed)

Reply:
Sure, done in https://github.com/brave/brave-core/pull/28568/commits/0b91dc905dc992273f19d70953c1a97ac45bedad